code/game/turfs/closed/_closed.dm

```diff
@@ -5,10 +5,6 @@
 	init_air = FALSE
 	rad_flags = RAD_PROTECT_CONTENTS | RAD_NO_CONTAMINATE
 	rad_insulation = RAD_MEDIUM_INSULATION
-	pass_flags_self = PASSCLOSEDTURF
```

Comment:
Why are you removing this?

Reply:
Stuff like this seems to happen in most of my PRs and I'm genuinely not sure why sometimes.